src/System.Runtime.Extensions/tests/System/BitConverter.cs

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Xunit;
+
+public static class BitConverterTests
+{
+    [Fact]
+    public static void DoubleToInt64Bits()
+    {
+        Double input = 123456.3234;
```

Comment:
Can you do a pass across these files and use the language aliases instead of the type names in the bodies of methods?